### PR TITLE
Added some additional file systems

### PR DESCRIPTION
**Pull request by martinvw**
As discussed via e-mail, or are you now merging manual?

### CODE REVIEW

**Comment by ralphje** [comment]
It will take some time to merge properly and review all changes. I certainly want to include lots of your commits. Some of my previous commits overlap with your work (some accidentally, e.g. I ran into the lack of FAT support before I noticed you included that too), but I'll try to sort those out in the merge.

I'm looking forward to expanding imagemounter drastically :smile:

File: imagemounter/disk.py [hunk 1]
```diff
@@ -471,6 +499,11 @@ def _mount_mmls_volumes(self):
                 volume.flag = 'unalloc'
             else:
                 volume.flag = 'alloc'
+                if ":" in slot:
+                    table, tslot = slot.split(':')
+                    volume.slot = int(table) * 4 + int(tslot)
```
Comment (ralphje) [inline imagemounter/disk.py]: @martinvw Do you have documentation or other insights for the magic you are performing here?

Comment (martinvw) [inline imagemounter/disk.py]: Hmm, partially :)

You can see an example of the use of collons here: http://digital-forensics.sans.org/blog/2010/01/11/using-image-offsets

00: ----- 0000000000 0000000000 0000000001 Primary Table (#0)
01: ----- 0000000001 0000000062 0000000062 Unallocated
02: 00:00 0000000063 0000064259 0000064197 DOS FAT12 (0x01)
03: 00:01 0000064260 0000273104 0000208845 DOS Extended (0x05)
04: ----- 0000064260 0000064260 0000000001 Extended Table (#1)
05: ----- 0000064261 0000064322 0000000062 Unallocated
06: 01:00 0000064323 0000273104 0000208782 DOS FAT16 (0x06)
07: 01:01 0000273105 0000433754 0000160650 DOS Extended (0x05)
08: ----- 0000273105 0000273105 0000000001 Extended Table (#2)
09: ----- 0000273106 0000273167 0000000062 Unallocated
10: 02:00 0000273168 0000433754 0000160587 NTFS (0x07)
11: 02:01 0000433755 0000481949 0000048195 DOS Extended (0x05)
12: ----- 0000433755 0000433755 0000000001 Extended Table (#3)
13: ----- 0000433756 0000433817 0000000062 Unallocated
14: 03:00 0000433818 0000481949 0000048132 Linux (0x83)
15: 03:01 0000481950 0000626534 0000144585 DOS Extended (0x05)
16: ----- 0000481950 0000481950 0000000001 Extended Table (#4)
17: ----- 0000481951 0000482012 0000000062 Unallocated
18: 04:00 0000482013 0000626534 0000144522 Linux Swap / Solaris x86 (0x82)
19: ----- 0000626535 0000629143 0000002609 Unallocated

Main problem is that only the primary table contains 4 records, all the other will contain at maximum 2, so the correct calculation is a little bit more complex and deserves a function on its own :)

File: imagemounter/util.py [hunk 2]
```diff
@@ -204,4 +213,20 @@ def force_clean(execute=True):
...
+    # VMKCore Diagnostic
+    elif guid == '8053279D-AD40-DB11-BF97-000C2911D1B8':
+        return 'vmkcore-diagnostics'
+    elif guid == '6A898CC3-1DD2-11B2-99A6-080020736631' or \
+         guid == 'C38C896A-D21D-B211-99A6-080020736631':
+        return 'zfs-member'
+    else:
+        if parser:
+            parser._debug('    GUID {} not yet supported'.format(guid))
+        return 'unknown'
```
Comment (martinvw) [inline imagemounter/util.py]: There are more complete list available on the internet, so we could add a lot more already as this code now a filesystem which is recognized but supported for mounting exists gracefully.

**Pull request merged**